### ISSUE TITLE
[Doc] Still using old MPLLIBS_STRING macro in the tutorial

### ISSUE THREAD

**Issue by ldionne** [boostorg/metaparse#2]
The tutorial is still using the `MPLLIBS_STRING` and `MPLLIBS_DEFINE_ERROR` macros instead of their `BOOST_xxx` counterpart. This should be fixed if the library makes it in.

**Comment by sabel83** [comment]
Yes, and it looks like the diagrams need to be updated as well. I'll fix these after the review (once the xxx part of the BOOST_xxx names have been chosen :))
